charts/ui-plugin-operator/templates/hardened.yaml

```diff
@@ -30,6 +30,7 @@ spec:
         args: ["-n", "{{ $ns }}"]
       {{- end }}
 ---
+{{ if .Capabilities.APIVersions.Has "policy/v1beta1/PodSecurityPolicies" }}
```

Comment:
It seems this service account is still used to patch other service accounts, according to: https://github.com/rancher/ui-plugin-operator/blob/a5e1382dd4c72ddf146968e690c55e02d36f4814/charts/ui-plugin-operator/templates/hardened.yaml#L25-L31

I think this conditional should be in line 45 instead (otherwise the SA will be missing permissions).

Reply:
alright, updated!